llrt_modules/Cargo.toml

```diff
@@ -19,6 +19,8 @@ all = [
   "process",
   "perf_hooks",
   "performance",
+  "exceptions",
+  "navigator",
```

Comment:
@richarddavison Those APIs are not node related, so thats why I think some kind of marker would be good

Reply:
Maybe something like
```rust
/// This is my module blablabla
///
/// # Reference
/// Source: node
```
It would be cool to produce a diagram of APIs supported from each reference implementation by scrapping the documentation.

Otherwise a macro like
```rust
#[ref(source: node)]
impl ModuleDef for MyModule {
  ...
}
```

Then we can generate an `impl ModuleSource for MyModule` and aggregate those easily.

Reply:
There are two kinds of APIs, node and browser. Most of the APIs on browsers are introduced in the form of global variables. Nodejs recommends modular usage.